configure.ac

```diff
@@ -1427,6 +1445,50 @@ AM_CONDITIONAL([EMBEDDED_UNIVALUE],[test x$need_bundled_univalue = xyes])
 AC_SUBST(UNIVALUE_CFLAGS)
 AC_SUBST(UNIVALUE_LIBS)
 
+dnl libmultiprocess library check
+
+libmultiprocess_found=no
+if test "x$with_libmultiprocess" = xyes || test "x$with_libmultiprocess" = xauto; then
+  if test "x$use_pkgconfig" = xyes; then
+    m4_ifdef([PKG_CHECK_MODULES], [PKG_CHECK_MODULES([LIBMULTIPROCESS], [libmultiprocess], [
+       libmultiprocess_found=yes;
+       libmultiprocess_prefix=`$PKG_CONFIG --variable=prefix libmultiprocess`;
+    ], [true])])
+  fi
+elif test "x$with_libmultiprocess" != xno; then
+  AC_MSG_ERROR([--with-libmultiprocess=$with_libmultiprocess value is not yes, auto, or no])
+fi
+AC_SUBST(LIBMULTIPROCESS_CFLAGS)
+AC_SUBST(LIBMULTIPROCESS_LIBS)
```

Comment:
In this PR these variables are not used in `Makefile`s.

But in #10102 one can see:
https://github.com/bitcoin/bitcoin/blob/aa38570fa5f1ceee5883a8170da60206051e05fd/src/Makefile.am#L597
https://github.com/bitcoin/bitcoin/blob/aa38570fa5f1ceee5883a8170da60206051e05fd/src/Makefile.am#L649

Is it intended?

Reply:
> If it's a concern about clarity, is there something beyond these two lines of code that is confusing?

Yes, it is about clarity. Thank you for your patient clarification regarding future changes.

Reply:
> Yes, it is about clarity. Thank you for your patient clarification regarding future changes.

Thank you for reviewing and testing and finding things to fix! I'd very much like to add comments to the build files to document things which are surprising. But I can't do that without knowing which specific things are surprising, so the more context provided in review comments, the better!